sdk/storage/Azure.Storage.Files/src/Models/FileRange.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Globalization;
+using System.Xml.Linq;
+
+namespace Azure.Storage.Files.Models
+{
+    public partial struct FileRange
+    {
+        /// <summary>
+        /// Deserializes XML into a new Range instance.
+        /// </summary>
+        /// <param name="element">The XML element to deserialize.</param>
+        /// <returns>A deserialized Range instance.</returns>
+        internal static FileRange FromXml(XElement element)
```

Comment:
This is clearly a hack because generator doesn't support structs in lists. Tell me if you want to try and fix it.